src/models/devices.ts

```diff
@@ -72,3 +75,12 @@ export function deleteDevice(id: string) {
 		},
 	});
 }
+
+export async function takeNFromDeviceHistory(device_uuid: string, n: number) {
```

Comment:
The function naming is confusing as looks like a typo of takenFrom....

How about nMostRecentLocations, or simple mostRecentLocations?